src/Microsoft.ML.FastTree/TreeTrainersCatalog.cs

```diff
@@ -125,23 +121,19 @@ public static class TreeExtensions
         }
 
         /// <summary>
-        /// Predict a target using a decision tree binary classification model trained with the <see cref="FastTreeBinaryClassificationTrainer"/>.
+        /// Predict a target using a decision tree binary classification model trained with the <see cref="RegressionGamTrainer"/>.
         /// </summary>
-        /// <param name="ctx">The <see cref="BinaryClassificationContext"/>.</param>
+        /// <param name="ctx">The <see cref="RegressionContext"/>.</param>
         /// <param name="label">The label column.</param>
         /// <param name="features">The features column.</param>
         /// <param name="weights">The optional weights column.</param>
-        /// <param name="numTrees">Total number of decision trees to create in the ensemble.</param>
-        /// <param name="numLeaves">The maximum number of leaves per decision tree.</param>
         /// <param name="minDatapointsInLeafs">The minimal number of datapoints allowed in a leaf of the tree, out of the subsampled data.</param>
```

Comment:
>minDatapointsInLeafs [](start = 25, length = 20)

I sense inconsistent naming

Reply:
enlighten me? 

---
In reply to: [231226770](https://github.com/dotnet/machinelearning/pull/1524#discussion_r231226770) [](ancestors = 231226770)

Reply:
MinDocsPerLeaf, MinDocumentsPerLeaf, MinDatapointsInLeafs

Also 'leaves', not 'leafs'

---
In reply to: [231232309](https://github.com/dotnet/machinelearning/pull/1524#discussion_r231232309) [](ancestors = 231232309,231226770)

Reply:
oh lol. Apparently it's been like this for a long time. 

---
In reply to: [231367379](https://github.com/dotnet/machinelearning/pull/1524#discussion_r231367379) [](ancestors = 231367379,231232309,231226770)